app/frontend/app/controllers/pro.js

```diff
@@ -104,7 +103,9 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
         key: PreloadStore.get('stripe_key'),
         name: "Hummingbird PRO",
         description: this.get('selectedPlan.name'),
-        token: tokenCallback
+        token: tokenCallback,
+        amount: parseInt(this.get('selectedPlan.amount') * 100),
```

Comment:
Just tested and it's not needed at all. `this.get('selectedPlan.amount') * 100` works, without issue.